src/vue/backend/src/mixins/standaloneSettingsActions.js

```diff
@@ -0,0 +1,81 @@
+export default {
+    data () {
+        return {
+            formDisabled: false
+        }
+    },
+    methods: {
+        async loadData (repo, path = '') {
+            this.formDisabled = true
+            try {
+                const { data } = await repo.get(path)
+                this.updateState(data)
+            } catch (error) {
+                this.errorAction()
+            }
+            this.formDisabled = false
+        },
+        updateState (data) {
+            for (const key in data) {
+                this.$store.dispatch(`${this.module}/setItem`, { key, value: data[key] })
+            }
+        },
+        scrollToTop () {
+            window.scrollTo({
+                top: 0,
+                left: 0,
+                behavior: 'smooth'
+            })
+        },
+        saveAction () {
+            this.updateState(this.formChanges)
+            this.formChanges = {}
+            this.scrollToTop()
+            this.$store.dispatch('alerts/setItem', { key: 'notification', value: { show: true, error: false, text: 'Changes Saved' } })
+            setTimeout(() => {
+                this.$store.dispatch('alerts/setItem', { key: 'notification', value: { show: false, error: false, text: 'Changes Saved' } })
```

Comment:
Should the `text` be cleared out here as well, or omitted?

Same question on line 43 with `error`

Reply:
Yeah! I'll update